README.rst

```diff
@@ -1001,6 +1093,67 @@ For example, sending recovery notifications.
 This signal is emitted when the threshold value of a ``Metric`` defined in
 alert settings is crossed.
 
+``pre_metric_write``
+~~~~~~~~~~~~~~~~~~~~
+
+**Path**: ``openwisp_monitoring.monitoring.signals.pre_metric_write``
+
+**Arguments**:
+
+- ``metric``: ``Metric`` object whose data shall be stored in timeseries database
+- ``values``: metric data that shall be stored in the timeseries database
+
+This signal is emitted for every metric before the write operation is sent to
+the timeseries database.
+
+``post_metric_write``
+~~~~~~~~~~~~~~~~~~~~~
+
+**Path**: ``openwisp_monitoring.monitoring.signals.post_metric_write``
+
+**Arguments**:
+
+- ``metric``: ``Metric`` object whose data is being stored in timeseries database
+- ``values``: metric data that is being stored in the timeseries database
+
+This signal is emitted for every metric after the write operation is sent to the
+timeseries database. Since the write operation is run in the background and may be
+retried on failures, using the ``post_metric_write`` signal does not guarantee the
+receiver function will be called after the data has been written, only after the
+request to write it has been sent.
+
+``health_status_changed``
```

Comment:
this is the most important signal, let's move it up to the first signal listed, pre metric and post metric write should go last.